pennylane/devices/tf_ops.py

```diff
@@ -191,3 +191,108 @@ def CRot(a, b, c):
         :math:`|0\rangle\langle 0|\otimes \mathbb{I}+|1\rangle\langle 1|\otimes R(a,b,c)`
     """
     return tf.linalg.diag(CRZ(c)) @ (CRY(b) @ tf.linalg.diag(CRZ(a)))
+
+
+def DoubleExcitation(phi):
+    r"""

```

Comment:
Side-note: formatting of the docstring is required for the docs too, not just for cosmetics:
https://pennylane--1123.org.readthedocs.build/en/1123/code/api/pennylane.devices.tf_ops.DoubleExcitation.html?highlight=doubleexcitation#pennylane.devices.tf_ops.DoubleExcitation